packages/fork-choice/src/forkChoice/forkChoice.ts

```diff
@@ -767,7 +767,7 @@ export class ForkChoice implements IForkChoice {
     return blocksAtSlot;
   }
 
-  /** Returns the distance of common ancestor of nodes to newNode. Returns null if newNode is descendant of prevNode */
+  /** Returns the distance of common ancestor of nodes to the max of the newNode and the prevNode. Returns null if newNode is descendant of prevNode */
```

Comment:
Seems this null comment is stale. Can you just remove it?

Reply:
Line 781 suggests the noCommonAncestor error code is equivalent to a null response which is why I left it

Reply:
I think that comment was from before we returned a `AncestorResult`.